src/index.js

```diff
@@ -16,3 +33,70 @@ function toggleMenu() {
 }
 
 hamburgerMenu.addEventListener('click', toggleMenu);
+
+let currentProfile = document.querySelector(
+	'.profile--selected'
+);
+const orgName = document.getElementById('about__name');
+const orgAbout = document.getElementById('about__desc');
+const orgPhoto = document.getElementById('about__photo');
+
+const orgProfiles = document.querySelectorAll('.profile');
+
+// Add event listener to profiles
+for (let i = 0; i < orgProfiles.length; i++) {
+	orgProfiles[i].addEventListener('click', selectProfile);
+	orgProfiles[i].addEventListener('keypress', (e) => {
+		if (handleClick(e)) {
+			selectProfile(e);
+		}
+	});
+}
+
+// Function to select organizer profile -- updates from organizers data object
+function selectProfile(e) {
+	let currentProfileId = e.currentTarget.id;
+
+	if (organizers[currentProfileId]) {
```

Comment:
In places where I reuse a value from an array over and over I like to create a variable. You could probably do something like this, and then check for null and swap out the references below: `const selectedOrganizer = organizers[currentProfileId];` How's that sound to you?

Reply:
Sounds good. Modified the code.